docs/actions-reusable.md

```diff
@@ -0,0 +1,63 @@
+Like [scripted actions](actions-scripts.md), reusable actions are logical units of analytic code.
+However, whereas the source code for a scripted action must be copied from study to study, the source code for a reusable action is shared by different studies.
+
+## Running reusable actions
+
+You can browse existing reusable actions at <https://actions.opensafely.org>.
+Although each is different, they have a common API.
+Consider the following extract from a study's *project.yaml*:
+
+```yaml
+actions:
+  generate_my_cohort:
+    # ...
+
+  run_a_reusable_action:
+    # The reusable action is called "a_reusable_action".
+    # We will run version "v1.0.0".
+    run: a_reusable_action:v1.0.0
```

Comment:
Could this include an argument or two?  And maybe a demonstration of the `config:` stuff?

Reply:
I've added an argument and a configuration option to the extract from a study's *project.yaml*. I'm keen to avoid using specific reusable actions as examples at this stage, because there's only `safetab` and `safetab` needs attention (see above).